NDTensors/src/dense/tensoralgebra/contract.jl

```diff
@@ -377,10 +377,16 @@ function _contract!(
   if props.permuteC
     # if we are computing C = α * A B + β * C
     # we need to make sure C is permuted to the same 
-    # ordering as A B
+    # ordering as A B which is the inverse of props.PC
     if β ≠ 0
-      pC = NTuple{NB,Int}(props.PC)
-      CM = reshape(permutedims(CT, pC), (props.dleft, props.dright))
+      pC = NTuple{NC,Int}(props.PC)
```

Comment:
The field `PC` is already a Tuple so this conversion shouldn't be necessary: https://github.com/ITensor/ITensors.jl/blob/e4779bdb8c2760028cea727a60ac602ed76930e0/NDTensors/src/tensoralgebra/contraction_logic.jl#L145

Reply:
Okay, I just followed what was already in the code. I removed all of the definitions of `p_ = NTuple{N_, Int}(props.P_` where `_` is `A, B, or C`

Reply:
Thanks. I think it was like that because at some point those fields of `ContractionProperties` had a different type so a conversion was necessary, but the rest of the code wasn't updated accordingly.